### PR TITLE
KDW-2458: Production k8s and readme update

### PR DESCRIPTION
**Pull request by caberson**
https://kargo1.atlassian.net/browse/KDW-2657

Production bot is deployed.  Have switched Slack app and the one Looker report to point to the production instance. I will turn off the dev version after a day.

### CODE REVIEW

**Review by chaithra-yenikapati — APPROVED** [review]
👍

File: k8s/dev/deployment.yaml [hunk 1]
```diff
@@ -72,6 +72,26 @@ spec:
           resources: {}
           terminationMessagePath: /dev/termination-log
           terminationMessagePolicy: File
+          livenessProbe:
+            httpGet:
+              path: /health_check
```
Comment (transistorize) [inline k8s/dev/deployment.yaml:77]: nice! I need a tutorial on liveness and readiness probles

File: k8s/README.md [hunk 2]
```diff
@@ -4,3 +4,29 @@ We forked the lookerbot code base from Looker's GitHub repository and set it up
...
+  ```bash
+  $ kubectl apply -f prod --context=production.us-east-1.kops.kargo.com --namespace=lookerbot 
+  ```
+
+* Development  
+  ```bash
+  $ kubectl apply -f prod --context=staging.us-east-1.kops.kargo.com --namespace=lookerbot-dev 
+  ```
+
+If existing secrets need update, you can use `template/secrets-template.yaml` as a template.  Update all the `FIXME` values.  Then apply the updated file to the applicable environment.
```
Comment (transistorize) [inline k8s/README.md]: If existing secrets need `an` update

File: k8s/README.md [hunk 3]
```diff
@@ -4,3 +4,29 @@ We forked the lookerbot code base from Looker's GitHub repository and set it up
...
+
+After you apply a deployment, you should use either Papertrail or the following command to verify the deployment was successful.
+```bash
+$ kubectl get pods -w --namespace=lookerbot --context=production.us-east-1.kops.kargo.com
+```
+
+Ways to quickly check if the production Looker bot is active.  Note that unless you've confirmed the Looker bot you deployed was made active, the following methods may just mean the last active Looker bot is still running:
```
Comment (transistorize) [inline k8s/README.md]: First sentence is a bit awkward, perhaps better as a subheading?

**Review by transistorize — CHANGES_REQUESTED** [review]

**Comment by caberson** [comment]
@transistorize, made the changes to README.

**Review by transistorize — APPROVED** [review]

**Pull request merged**